Dockerfile

```diff
@@ -19,6 +20,7 @@ RUN cpanm \
 
 #Copy source code
 COPY cloc /usr/src/
+RUN find /usr/src/ -type f -exec dos2unix {} \;
```

Comment:
When I built this on Windows, running the perl-script failed because of Windows-line endings in the script.